platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorStyledTextView.kt

```diff
@@ -158,6 +159,12 @@ open class EditorStyledTextView : AppCompatTextView {
         htmlConverter?.fromHtmlToSpans(htmlText)?.let { setText(it, BufferType.SPANNABLE) }
     }
 
+    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
+
+        layout?.let { onTextLayoutChanged?.invoke(it) }
```

Comment:
Here the text has already been laid out so we can return `this.layout` through the callback.